vhd/lib/libvhd.c

```diff
@@ -744,7 +726,7 @@ vhd_bitmap_test(vhd_context_t *ctx, char *map, uint32_t block)
 {
 	if (vhd_creator_tapdisk(ctx) &&
```

Comment:
These should probably now be fatal errors, if there are any VHDs still out there which were created with the broken 0.1 version prior to May 2008 then they're long past due being removed.

Reply:
now made to abort